File: CommonModule/src/main/java/com/qihuan/commonmodule/utils/ToastUtils.java
package com.qihuan.commonmodule.utils;

import android.app.Application;
import android.support.v4.content.ContextCompat;
import android.widget.Toast;

import com.qihuan.commonmodule.R;

import es.dmoral.toasty.Toasty;

/**
 * ToastUtils
 * Created by Qi on 2017/6/22.
 */

public class ToastUtils {

    public static void info(String content) {
        Toasty.info(AppUtils.getContext(), content, Toast.LENGTH_SHORT).show();
    }

    public static void error(String content) {
        Toasty.error(AppUtils.getContext(), content, Toast.LENGTH_SHORT).show();
    }

    public static void success(String content) {
        Toasty.success(AppUtils.getContext(), content, Toast.LENGTH_SHORT).show();
    }

    public static void warning(String content) {
        Toasty.warning(AppUtils.getContext(), content, Toast.LENGTH_SHORT).show();
    }

    public static void init(Application application) {
        Toasty.Config
                .getInstance()
                .setInfoColor(ContextCompat.getColor(application, R.color.grey))
                .apply();
    }
}

File: ModuleCommon/src/main/java/com/qihuan/commonmodule/utils/ToastUtils.kt
package com.qihuan.commonmodule.utils

import android.content.Context
import android.widget.Toast
import es.dmoral.toasty.Toasty

/**
 * ToastUtils
 * Created by Qi on 2017/6/22.
 */

fun Context.toastInfo(content: String) {
    Toasty.info(this, content, Toast.LENGTH_SHORT).show()
}

fun Context.toastError(content: String) {
    Toasty.error(this, content, Toast.LENGTH_SHORT).show()
}

fun Context.toastSuccess(content: String) {
    Toasty.success(this, content, Toast.LENGTH_SHORT).show()
}

fun Context.toastWarning(content: String) {
    Toasty.warning(this, content, Toast.LENGTH_SHORT).show()
}